Tests/Transformer/ModelToElasticaAutoTransformerTest.php

```diff
@@ -473,6 +483,54 @@ public function testThatNestedObjectsDontNeedAnIdentifierField()
         ), $data['subWithoutIdentifier']);
     }
 
+    public function testNestedTransformHandlesSingleObjects()
+    {
+        $transformer = $this->getTransformer();
+        $document    = $transformer->transform(new POPO(), array(
+            'upper' => array(
+                'type' => 'nested',
+                'properties' => array('name' => null)
```

Comment:
@XWB ``name: ~`` in YAML does not mean a ``'~'`` string after parsing. An unquoted ``~`` value is just an alternate syntax for ``null``. Is it your misunderstanding ?